### DISCUSSION TITLE
Create a Snap Target layout editor

### DISCUSSION THREAD

**Discussion by lpridpath** [rxhanson/RectanglePro-Community#242] [Ideas]
Hey Ryan! Loving Rectangle Pro. I posted a Q&A a while back inquiring about similar functionality that I use with FanzyZones on my windows machine. You responded and mentioned this could be done utilizing Snap Areas and custom shortcuts. After trying it out I can't seem to help that it's a little non intuitive or clumsy. If there are docs surrounding how to do so please let me know and I'd be happy to take a look!

I was wondering your thoughts about including a simple snap target builder UI or layout editor. I'm not sure exactly the best words to use here to try and convey what I am meaning, but I essentially mean what's showcased in [this video](https://www.youtube.com/watch?v=rTtGzZYAXgY&ab_channel=MikeHarsh) tutorial at 1:30 in.

Thanks in advance. I appreciate your craft and the hard work you put in to make this such a powerful and essential app.

Best

**Comment by rxhanson** [comment]
Thanks for the feedback! As you might suspect, this is on my todo list. It's kind of a big task, so it is waiting on when I get a good block of time to focus on it.

**Reply by rxhanson** [reply]
(It's also on my todo list to make tutorials on what currently exists)

The only tutorial doc for building custom shortcuts is here: https://github.com/rxhanson/RectanglePro-Community/discussions/5

**Reply by lpridpath** [reply]
Makes sense! Just glad to hear it's on the roadmap. If you're like me that list grows longer everyday and is tough to get to everything when you'd like; so I totally get that. I'll patiently wait with excitement. 

Thanks for the link to the tutorial doc. If you do setup something to [github wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis) documentation I'd love to contribute.
